recipes/hdf5/meta.yaml

```diff
@@ -26,6 +26,43 @@ requirements:
   run:
     - zlib
 
+test:
+  commands:
+    # Verify CLI tools.
+    - gif2h5 --help
+    - h52gif --help
+    - h5c++ --help
+    - h5cc --help
+    - h5copy --help
+    - h5debug --help
+    - h5diff --help
+    - h5dump --help
+    - h5import --help
+    - h5jam --help
+    - h5ls --help
+    - h5mkgrp --help
+    - h5perf_serial --help
+    - h5redeploy --help
+    - h5repack --help
+    - h5repart --help
+    - h5stat --help
+    - h5unjam --help
```

Comment:
Yeah, some of these don't have `--help` so we switched to using `command -v` on UNIX platforms and `where` on Windows.